src/Components/StyledPostArticle.js

```diff
@@ -6,5 +6,7 @@ const CardArticle = styled.article`
 	margin: 10px;
 	border-radius: 5px;
 	box-shadow: 2px 2px 8px grey;
+	width: 350px;
+	height: 300px;
 `;
 export { CardArticle };
```

Comment:
As we only want to export ONE thing per file, you could make this a default export. 🙃